.cloudbuild/ci/integration-tests.yaml

```diff
@@ -0,0 +1,29 @@
+steps:
+  # Run non-root integration tests. The root user is selected by docker image.
+  - name: us-docker.pkg.dev/ci-account/teleport/buildbox-root:v0.0.7
```

Comment:
Didn't we change those labels to `v0.x.0`? Maybe it makes sense to label as `buildbox-root:v0.1.0-teleport7` or something like that, to keep versions separate?

Reply:
Yeah, but then I realized I had to create compatible buildboxes for old branches. I like the idea of tagging them `teleport7` rather than `v0.0.7`. Will rebadge and update.

Reply:
Tagged as `teleport7-v0.1.0`

Reply:
Looks good, thanks!